packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -120,8 +120,18 @@ class CupertinoSwitch extends StatefulWidget {
     super.key,
     required this.value,
     required this.onChanged,
-    this.activeColor,
-    this.trackColor,
+    @Deprecated(
+      'Use activeTrackColor instead. '
+      'This feature was deprecated after v3.24.0-0.0.pre.'
```

Comment:
Note to self/ other reviewers: check that this is still the up to date message before merging

Reply:
Because I often forget where to check for this: https://github.com/flutter/flutter/tags

Reply:
Can you check and update this version number in the deprecation messages? I think we are really close now.